ncs/Kconfig

```diff
@@ -26,66 +26,6 @@ config SUIT_ENVELOPE_OUTPUT_MPI_MERGE
   bool "Merge MPI files into final SUIT_ENVELOPE_OUTPUT_ARTIFACT"
   default y
 
-menuconfig SUIT_ENVELOPE
-    bool "Create SUIT envelope"
-    help
-        Enable DFU SUIT envelope creation
-    default y if SSF_SUIT_SERVICE_ENABLED && SOC_NRF54H20_CPUAPP
-
-if SUIT_ENVELOPE
-
-config SUIT_ENVELOPE_SIGN
-    bool "Sign created SUIT envelope"
-    help
-        Sign created SUIT envelope by external script
-    default n
-
-config SUIT_ENVELOPE_SEQUENCE_NUM
-    int "Sequence number of the generated SUIT manifest"
-    range 0 2147483647
-    default 1
-
-config SUIT_ENVELOPE_ROOT_TEMPLATE
```

Comment:
Did you try to use change this from for example smp_transfer/sysbuild/smp_transfer.conf? It does not seem to work

Reply:
I have aligned the smp_transfer sample now.

Reply:
Ok, nevermind